libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -65,7 +65,8 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Param: AUTO_SWITCH
     // @DisplayName: Automatic Switchover Setting
     // @Description: Automatic switchover to GPS reporting best lock
-    // @Values: 0:Disabled,1:Enabled
+    // @Values: 0:Disabled,1:Hard,2:Soft
+    // @RebootRequired: True
     // @User: Advanced
     AP_GROUPINFO("AUTO_SWITCH", 3, AP_GPS, _auto_switch, 1),
```

Comment:
Ok, if that's what the majority prefers. My issue is that UseBest doesn't always uses the best one (and that's one of the reasons we've recommended against using it).